README.md

```diff
@@ -59,7 +59,7 @@ would suggest that you upgrade to a browser that supports Service Workers (Chrom
 
 ### Officially supported platforms
 
-- <img src="images/firefoxbrowser-color.svg" width="20" /> Mozilla Firefox >=56 (as an extension): [Mozilla Add-ons Store](https://addons.mozilla.org/fr/firefox/addon/kiwix-offline/)
+- <img src="images/firefoxbrowser-color.svg" width="20" /> Mozilla Firefox >=56 (as an extension): [Mozilla Add-ons Store](https://addons.mozilla.org/en-US/firefox/addon/kiwix-offline/)
```

Comment:
Thanks for this PR, but you should choose a language-neutral link, so that users can view the extension in their own language. I suggest changing this to `https://addons.mozilla.org/firefox/addon/kiwix-offline/`, as this seems to redirect to the user's underlying browser language.

Reply:
@Jaifroid Yes that's much better